lib/portlayer/storage/vsphere/image.go

```diff
@@ -395,7 +397,12 @@ func (v *ImageStore) scratch(op trace.Operation, storeName string) error {
 
 	// Write the metadata to the datastore
 	metaDataDir := v.imageMetadataDirPath(storeName, portlayer.Scratch.ID)
-	if err := writeMetadata(op, v.ds, metaDataDir, nil); err != nil {
+	scratchMetaData := &metadata.ScratchMetaData{
+		ScratchVersion: feature.ScratchBaseStructureVersion,
+	}
+	meta := make(map[string][]byte)
+	meta["metaData"] = []byte(fmt.Sprintf("%+v", scratchMetaData))
```

Comment:
Suggest using a constant for this key, a meaningful key as everything stored in this fashion is metadata, and input closer to the namespaced style used with WriteImage, e.g
```
portlayer.storage.scratch
```